packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/MediaLibraryUtils.java

```diff
@@ -358,8 +369,40 @@ static void getExifFullInfo(ExifInterface exifInterface, Bundle response) {
     response.putParcelable("exif", exifMap);
   }
 
+  //  reference: https://developer.android.com/training/data-storage/shared/media#location-info-photos
+  @RequiresApi(api = Build.VERSION_CODES.Q)
+  static void getExifLocationForUri(ContentResolver contentResolver, Uri photoUri, Bundle asset) throws Exception {
```

Comment:
does this throw general `Exception` or maybe some specific one? It would be good to narrow down the exception range because error handling in MediaLib is already a bit confusing (regarding the real source of error).

_Hint: when you hover and click on `throws Exception` (the Exception word) - Android Studio highlights all calls which throw that exception (or its subclasses)_